dist/api_data.py

```diff
@@ -857,6 +860,24 @@ def __cmp__(self, other):
         if non-empty, checkpoint the list of objects''', type='list'),
 ]),
 
+'WT_SESSION.snapshot' : Method([
+    Config('drop', '', r'''
+            if non-empty, specifies which snapshots to drop. Where a group
+            of snapshots are being dropped, the order is based on snapshot
+            creation order not alphanumeric name order''',
+        type='category', subconfig=[
+        Config('names', '', r'''
+            drop specific named snapshots''', type='list'),
+        Config('to', 'true', r'''
+            drop all snapshots up to and including the specified name.
+            If the special "all" name is given all snapshots will be
+            dropped.'''),
```

Comment:
Given that we have a subconfig for "drop" already, should we make "all" a separate config key, rather than requiring the clunky "to=all"?